python/lsst/daf/butler/datastores/chainedDatastore.py

```diff
@@ -456,3 +457,40 @@ def transfer(self, inputDatastore, ref):
         assert inputDatastore is not self  # unless we want it for renames?
         inMemoryDataset = inputDatastore.get(ref)
         return [datastore.put(inMemoryDataset, ref) for datastore in self.datastores]
+
+    def validateConfiguration(self, *entities, logFailures=False):
+        """Validate some of the configuration for this datastore.
```

Comment:
I think we starting to use more of

        # Docstring inherited from Datastore.validateConfiguration

style of comments instead of copying whole docstrings from base classes?

Reply:
Agreed, but here the notes section is different because it explains that each datastore is queried in turn.

Reply:
I wish we could extend inherited docstrings. Or could we do it with something like

    See `BaseClass.method` for description of parameters and return types.

    Notes
    -------
    Some additional info

?